components/locid/src/extensions/private/mod.rs

```diff
@@ -132,6 +150,10 @@ impl Deref for Private {
     type Target = [Key];
 
     fn deref(&self) -> &Self::Target {
-        &self.0
+        if let Some(ref data) = self.0 {
+            data
+        } else {
+            &[]
+        }
```

Comment:
Suggestion: `self.0.as_ref().unwrap_or(&[])`

Reply:
Thanks for the suggestion, but in this case I find the explicit if/else more readable.  I get confused when `as_ref()` is used on an Option.